src/App.tsx

```diff
@@ -1,204 +1,99 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { TodoMain } from './components/TodoMain';
+import { TodoHeader } from './components/TodoHeader';
+import { FilterType } from './types/FilterType';
+import { TodoNotification } from './components/TodoNotification';
+import { TodoFooter } from './components/TodoFooter';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { ErrorType } from './types/ErrorType';
+
+const USER_ID = 12031;
+
+const filterTodos = (sourceTodos: Todo[], filterBy: FilterType) => {
+  let filteredTodos: Todo[];
+
+  switch (filterBy) {
+    case FilterType.Active:
+      filteredTodos = sourceTodos.filter(todo => !todo.completed);
+      break;
+
+    case FilterType.Completed:
```

Comment:
AFAIK, better practice will name enum's items in UPPER_CASE
btw, it's just recommendation =)

Reply:
@Anton-Kuchmasov usually, each developer writes it in his own way, so the current version is also suitable )